portfolio/src/main/java/com/google/sps/servlets/OfficeGame.java

```diff
@@ -55,7 +61,20 @@
   /** Is true or false based on whether the user got the last question right */
   private boolean lastSubmission = true;
 
-  private int score = 0;
+  private long score = 0;
+
+  private DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+
+  private long highScore = -1;
```

Comment:
move this `highScore` above `datastore`. Variables of the same type should stay together.